satellite/satellitedb/projectaccounting.go

```diff
@@ -426,15 +426,15 @@ func (db *ProjectAccounting) GetBucketTotals(ctx context.Context, projectID uuid
 	bucketsQuery := db.db.Rebind(`SELECT DISTINCT bucket_name
 		FROM bucket_bandwidth_rollups
 		WHERE project_id = ? AND interval_start >= ? AND interval_start <= ?
-		AND CAST(bucket_name as TEXT) LIKE ?
+		AND bucket_name LIKE ?
```

Comment:
is this query safe from sql injection?

Reply:
we should add some tests that checks it.
look how it was tested in project_members
https://github.com/storj/storj/blob/master/satellite/console/projectmembers_test.go#L87